Assets/Scripts/OrchestraArmy/Room/RoomController.cs

```diff
@@ -221,5 +235,36 @@ public void OnEvent(RoomDoorRightEvent invokedEvent)
         {
             ChangeCurrentRoom(DoorDirection.Right);
         }
+
+        private void Update()
```

Comment:
Persoonlijk zou ik dit in een couratine doen. Kun je makkelijk een x aantal seconden wachten tussen de stappen, op dit moment blijft het death screen 3 frames zichtbaar, dat gaat wel super snel ;)

Reply:
standaard 3 seconden deathScreen (ook voor snelle computers)